app/commands/contributor_team/add_track_maintainer.rb

```diff
@@ -0,0 +1,27 @@
+class ContributorTeam
+  class AddTrackMaintainer
+    include Mandate
+
+    initialize_with :user, :track, :attributes
+
+    def call
+      team = ContributorTeam.find_by!(track: track, type: :track_maintainers)
+      ContributorTeam::Membership::CreateOrUpdate.(user, team, attributes)
+
+      Github::Team::AddMember.(team.github_name, user.github_username)
```

Comment:
This should probably move inside the `ContributorTeam::Membership::CreateOrUpdate`